File: src/main/java/itmo/is/project/mapper/trade/TradeMapper.java
package itmo.is.project.mapper.trade;

import itmo.is.project.dto.trade.TradeDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.mapper.user.UserMapper;
import itmo.is.project.model.trade.Trade;
import org.mapstruct.Mapper;

@Mapper(uses = {UserMapper.class, TradeItemMapper.class})
public interface TradeMapper extends EntityMapper<TradeDto, Trade> {
}

File: src/main/kotlin/itmo/isproject/trade/mapper/TradeMapper.kt
package itmo.isproject.trade.mapper

import itmo.isproject.trade.dto.TradeDto
import itmo.isproject.shared.common.mapper.EntityMapper
import itmo.isproject.shared.user.mapper.UserMapper
import itmo.isproject.trade.model.Trade
import org.mapstruct.Mapper

@Mapper(uses = [UserMapper::class, TradeItemMapper::class])
interface TradeMapper : EntityMapper<TradeDto, Trade>
